cl/recap/models.py

```diff
@@ -211,6 +212,53 @@ def file_contents(self):
     def print_file_contents(self):
         print(self.file_contents)
 
+class EmailProcessingQueue(AbstractDateTimeModel):
+    uploader = models.ForeignKey(
+        User,
+        help_text="The user that uploaded the item to RECAP.",
+        related_name="recap_email_processing_queue",
+        on_delete=models.CASCADE,
+        null=True,
+    )
+    court = models.ForeignKey(
+        Court,
+        help_text="The court where the upload was from",
+        related_name="recap_email_processing_queue",
+        on_delete=models.CASCADE,
+        null=True,
```

Comment:
We could pretty easily make this a required field. The sender of the email will presumably be something like:

    ecf-notices@cand.uscourts.gov

If we parse that in the labda, we can get the court ID (`cand`), and then send that as part of our POST to the API. We would have to include `cl.lib.pacer.map_pacer_to_cl` into the lambda, but I assume it'd be straightforward otherwise.

Reply:
Oh, I meant to say that the reason this is useful is because things tend to break on a court-by-court basis. If your parser is breaking and you can't filter by court (because your parser is broken), it gets annoying to debug. OTOH, if you parse for the court earlier, you can see which courts are broken and address them more easily.

Reply:
@mlissner Makes sense, this is more logic than I would've expected on the Lambda side. Not a problem, but while dealing with that, I may want to come back on the next PR.